jep/0000/README.adoc

```diff
@@ -0,0 +1,256 @@
+= JEP-0000: Use UTF-8 for Pipeline build logs
+:toc: preamble
+:toclevels: 3
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+.Metadata
+[cols="2"]
+|===
+| JEP
+| 0000
+
+| Title
+| Use UTF-8 for Pipeline build logs
+
+| Sponsor
+| link:https://github.com/jglick[Jesse Glick]
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Standards
+
+| Created
+| 2018-06-22
+
+| BDFL-Delegate
+| :bulb: Link to github user page :bulb:
```

Comment:
Who should be the BDFL Delegate for this?

Reply:
@bitwiseman I'm willing to take it if KK approves that -- and have already given my approval for the JEP so if he approves my approval, then it's merge-able.